docs/pages/docs/index.mdx

```diff
@@ -9,22 +9,29 @@ import Features from "../../components/Features";
 
 <Features page="docs" detailed={false} />
 
-Turborepo is a blazing fast build system for JavaScript/TypeScript _monorepos_: codebases containing multiple
-projects, often using multiple frameworks, in a single, unified
-code repository.
+Turborepo is a blazing fast build system for JavaScript/TypeScript monorepos - codebases containing multiple projects in a single, unified repository.
 
-Monorepos have many advantages, but they require appropriate tooling to scale. Turborepo provides this tooling and more. It leverages advanced build system ideas and techniques to speed up development, but with a fraction of the configuration and complexity. Turborepo abstracts all the annoying configuration, scripts, and tooling into `turbo.json`, so you can focus on _what_ you're shipping instead of worrying about _how_ it'll get shipped.
+## The problem
 
-Most importantly, unlike other build systems, Turborepo is designed to be incrementally adopted, so you can gradually and partially add it to most codebases in a few minutes.
+Monorepos have many advantages - but **they struggle to scale**. Each repository you add has its own test suite, its own build process and its own pipeline. A single monorepo might have **hundreds of tasks to perform**.
```

Comment:
- "Each repository you add" is confusing and may lead people down the path of thinking of submodules. "Each project you add" may be better
- "its own build process and its own pipeline" seems redundant. Also we may to reserve "pipeline" for the concept Turbo wants to introduce, rather than how a team already uses it.
- Mixing second and third persons here. "you add" (2nd person) vs "a single mono repo might have" (subject is 3rd person / thing)

Reply:
Addressed